liota/lib/protocols/helix_protocol.py

```diff
@@ -31,7 +31,8 @@
 # ----------------------------------------------------------------------------#
 
 import logging
-
+import json
+import time
 
 PROTOCOL_VERSION = "2.7"
```

Comment:
Need to change protocol version at both ends, so that older clients know it is a failure due to protocol mismatch